projects/hslayers/src/components/utils/utils.service.ts

```diff
@@ -577,3 +548,44 @@ export function generateUuid(): string {
     return v.toString(16);
   });
 }
+
+/**
+ * Check if object is an instance of a class
+ * @param obj -
+ * @param type -
+ * @returns true when obj is an instance of type, false otherwise
```

Comment:
```suggestion
 * Check if object is an instance of a specific class
 * @param obj - any object to check
 * @param type - class type itself
 * @returns true when obj is an instance of the provided type, false otherwise
```